docs/topics/lombok.md

```diff
@@ -0,0 +1,149 @@
+[//]: # (title: Lombok compiler plugin)
+
+> The Lombok compiler plugin is [Experimental](components-stability.md). 
+> It may be dropped or changed at any time. Use it only for evaluation purposes. 
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+Since 1.5.20, Kotlin supports the experimental Lombok Kotlin compiler plugin. This plugin allows the use of
+Java's Lombok-generated declarations in mixed Kotlin/Java sources.
+If Java's Lombok-generated declarations are in another module than the Kotlin code that uses these declarations, 
+then you don't need to use this plugin.
+
+[Lombok](https://projectlombok.org/) does Java code generation itself, not the Lombok Kotlin compiler plugin. 
+So, you still need to configure Lombok as usual, when using this plugin. 
+Learn more about [how to make the plugin seeing Lombok's config](#defining-the-place-of-lombok-config).
+
+## Support of annotations
```

Comment:
I'd call this "Supported Lombok annotations"

Reply:
But will it then be valid to leave about Builder here?

Reply:
IIRC I've left a comment that I don't see it valid in this section in general.

Reply:
This one: https://github.com/JetBrains/kotlin-web-site/pull/2340#discussion_r653505151